git/delete_remote_branch_test.go

```diff
@@ -0,0 +1,45 @@
+package git_test
+
+import (
+	"strings"
+	"testing"
+)
+
+func TestDeleteRemoteBranch(t *testing.T) {
+	skipWithoutGit(t)
+
+	testRepoGit, testRepoDir, cleanup := createTestRepo(t)
+	defer cleanup()
+
+	featureBranchName := "feature"
+	testRepoGit("checkout", "-b", featureBranchName)
+
+	createFile(t, testRepoDir, foo)
+	testRepoGit("add", foo.Name)
+	commitToFixMessage := "Add foo"
+	testRepoGit("commit", "-m", commitToFixMessage)
+
+	// Checkout master because git by default doesn't allow pushing onto or
```

Comment:
the _pushing_ part was irrelevant here: https://github.com/salemove/github-review-helper/pull/28/commits/3e5f57bedb14ddf4f002c5141c33250dd9b72977